src/store/modules/orders/actions.ts

```diff
@@ -0,0 +1,192 @@
+import { OrderService } from "@/services/OrderService";
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import OrdersState from './OrdersState'
+import * as types from './mutation-types'
+import { hasError , showToast } from "@/utils";
+import { translate } from "@/i18n";
+import emitter from '@/event-bus'
+import router from "@/router";
+
+const actions: ActionTree<OrdersState , RootState> ={
+  async getOrder ({ commit, state }, payload) {
```

Comment:
Improve method name

Reply:
Improved method name to `getOpenOrders`.